docs/learning/presentation/RulesetVariables.md

```diff
@@ -1,86 +1,97 @@
 # Ruleset Variables
 
-> *User Settings* is a deprecated name of the *Ruleset Variables* concept.
-
-Presentation rulesets have a concept called *ruleset variables* whose purpose is
-to allow modifying ruleset behavior during the session without having to change
-the ruleset itself. The values of ruleset variables can be accessed using
-[ECExpressions](./ECExpressions.md#ruleset-variables-user-settings).
-
-## User-Controllable Variables
-
-> This feature is not fully supported yet - there is no UI component for
-> editing user-defined ruleset variables.
-
-There can be either internal variables or user-controllable ones. Internal
-variables are not visible to end users and can only be changed using API calls.
-User-controllable variables are defined in the ruleset with pre-defined default
-values and may be modified by end users through some UI component.
-
-## Examples
-
-Specifying user-controllable ruleset variable in a ruleset:
-
-```JSON
-{
-  "id": "ruleset_id",
-  "rules": [],
-  "vars": [{
-    "label": "Variables group label",
-    "vars": [{
-      "id": "boolean_var_id",
-      "type": "ShowHide",
-      "label": "Show something?"
-    }, {
-      "id": "string_var_id",
-      "type": "StringValue",
-      "label": "Filtering value"
-    }, {
-      "id": "int_var_id",
-      "type": "IntValue",
-      "label": "Minimum value"
-    }],
-  }]
-}
+> Ruleset Variables is a new name for now deprecated *User Settings* concept.
+
+Presentation rulesets have a concept called *ruleset variables* whose purpose is to allow modifying ruleset behavior during the session without having to change
+the ruleset itself. The values of ruleset variables can be accessed using [ECExpressions](./ECExpressions.md#ruleset-variables-user-settings) in:
+
+- Rule conditions ([example](#using-variables-in-rule-condition)).
+- Instance filters ([example](#using-variables-in-instance-filter)).
+- Some ECExpressions-based values, like [LabelOverride.label]($presentation-common) ([example](#using-variables-in-ecexpression-based-value)).
+
```

Comment:
This phrasing suggests that Ruleset Variables cannot be used in all ECExpressions. Is that the case?

Reply:
Removed `Some`

Reply:
Thanks